h2o-docs/src/product/data-science/eif.rst

```diff
@@ -0,0 +1,122 @@
+.. _isoforestextended:
+
+Extended Isolation Forest
+-------------------------
+
+Introduction
+~~~~~~~~~~~~
+
+Extended Isolation Forest algorithm generalizes its predecessor algorithm,
+the `Isolation Forest <if.html>`__. The original `Isolation Forest <if.html>`__ algorithm brings a
+brand new form of detection, although the algorithm suffers
+from bias coming from tree branching. Extension of the algorithm
+mitigates the bias by adjusting the branching,
+and the original algorithm becomes just a special case.
+
+The cause of the bias is that branching is defined by the similarity
+to BST. At each branching point the
+feature and the value are chosen; this introduces the
+bias since the branching point is parallel to one of the axes.
+The general case needs to define a random slope for each branching point.
+Instead of selecting the feature and value, it selects a random slope :math:`n` for
+the branching cut and a random intercept :math:`p`. The slope can
+be generated from :math:`\mathcal{N(0,1)` Gaussian distribution, and the
+intercept is generated from the uniform distribution with bounds coming
+from the sub-sample of data to be split. The branching criteria for the data
+splitting for a given point :math:`x` is as follows:
+
+.. math::
+    (x - p) � n < 0
+
+Tutorials and Blogs
+~~~~~~~~~~~~~~~~~~~
+
+The following tutorials are available that describe how to use Extended Isolation Forest. 
+
+- todo link to notebooks? Thesis? External Sources?
```

Comment:
You can add links to IPython notebooks here and the link to the thesis too. Or if you find some interesting blog posts about EIF, you can also leave the links here.

Reply:
I added a link to the thesis and notebook about IF and EIF comparison by the authors of the algorithm. I am not sure if the notebooks [here](https://github.com/valenad1/h2o-3/tree/PUBDEV-7138-extended-isolation-forest/h2o-py/demos/extisofor) are good enough to be in the documentation.